src/types.ts

```diff
@@ -37,12 +37,13 @@ declare module 'fastify' {
 }
 
 type IgnoredTypes = Promise<unknown> | ((...args: unknown[]) => unknown);
+type CdcDate = number | string;
```

Comment:
I would rather keep this as `number` and maybe cast on specific parts where `timestampz` is used, because this will introduce a bunch of places where it breaks types. cc: @rebelchris wdyt?

Reply:
It might not be obvious but in strict TS mode this PR adds another 100 type errors in the codebase.

Reply:
I rather would see a move from timestamptz to timestamp then trying to retro fix it with potential of breaking implementations now.

Not sure how everyone feels about that?

Reply:
I am up for that too, alternative is to just handle it in places where timestampz is used, we can explicit cast `value as unknown as string` and thats it (in case we don't want to complicate with DB migrations now).

Reply:
After checking the build, there seem to be no difference in terms of errors:

branch MI-504
![Screenshot 2024-08-27 at 5 04 31 PM](https://github.com/user-attachments/assets/91cb3a1e-df9f-4527-ad1d-5d3ad9bce3bc)

this branch. It actually fixed one more thing.
![Screenshot 2024-08-27 at 5 05 08 PM](https://github.com/user-attachments/assets/61317384-3305-443e-806a-c901efbee8e7)

Reply:
The branch was not just updated from the main, hence it looked like it introduced strict mode issues. Updated it and now it is even better.

Reply:
@sshanzel oh, my bad on the number of errors part then.

Reply:
@rebelchris since there was no particular reason for using `timestamptz`, I changed it to `timestamp`. We can probably reopen this PR if we still want the type change, it is still correct under certain circumstances.

Relevant change: https://github.com/dailydotdev/daily-api/pull/2151/commits/91a7d32f0e0338c23f7b862cb45c6dcbf1e5ee00